config/dataset/test_ci_segmentation_dataset.yaml

```diff
@@ -0,0 +1,17 @@
+# @package _global_
+dataset:
+  name:
+  modalities: RGB
```

Comment:
chosing modalities is not implemented. This should be tagged as a TODO for future PR.